flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/api/TableITCase.scala

```diff
@@ -119,7 +119,13 @@ class TableITCase(tableEnvName: String, isStreaming: Boolean) extends TestLogger
     assertEquals(ResultKind.SUCCESS_WITH_CONTENT, tableResult.getResultKind)
     val it = tableResult.collect()
     it.close()
-    assertEquals(JobStatus.CANCELED, tableResult.getJobClient.get().getJobStatus().get())
+    try {
+      assertNotEquals(JobStatus.RUNNING, tableResult.getJobClient.get().getJobStatus.get())
+    } catch {
+      // ignore the exception,
+      // because the MiniCluster maybe already been shut down when getting job status
+      case _: Throwable => None
```

Comment:
This also catch exception of `assertNotEquals` and makes `assertNotEquals` useless.